scrunch/tests/test_datasets.py

```diff
@@ -1986,6 +1986,178 @@ def test_recode_multiple_response(self):
             }
         })
 
+    def test_create_categorical_missing_case(self):
```

Comment:
I see you only have one test and you have added many conditions, looks like there's untested paths still? Let me know if they're already tested elsewhere in case it isn't necessary to add more scenarios.